doc/modules/model_evaluation.rst

```diff
@@ -1503,11 +1492,7 @@ the keyword argument ``multiclass`` to ``'ovo'`` and ``average`` to
 ``'weighted'``. The ``'weighted'`` option returns a prevalence-weighted average
 as described in [FC2009]_.
 
-|details-end|
-
-|details-start|
-**One-vs-rest Algorithm**
-|details-split|
+.. topic:: One-vs-rest Algorithm:
```

Comment:
I am so sorry! I may have not been clear enough :sweat_smile:

The references get broken whenever they point inside a dropdown. Cross-references from the dropdown to the outside (such as this one) are perfectly fine!

The underlying issue is that the dropdown has to be unfolded for mozilla to access the reference.

Reply:
No probs! I've updated the references to topic directives when required (assuming I understood correctly this time. If not, no problem and we'll keep modifying the rst)